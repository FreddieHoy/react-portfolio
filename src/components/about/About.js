import { Container } from 'components/common/Container';
import { mobileBreakPoint } from 'config/spacing';
import React from 'react';
import HeadShot from 'static/HeadShot.jpg';
import styled from 'styled-components';

import {
  AboutContentContact,
  AboutContentInterests,
  AboutContentMain
} from './AboutContent';
import { AboutTitle } from './AboutTitle';

const Image = styled.img`
  width: 100%;
  height: 100%;
  padding: 26px 0;

  @media (min-width: ${mobileBreakPoint}px) {
    width: 40%;
  }
`;

export const About = () => (
  <Container>
    <AboutTitle />
    <AboutContentMain />
    <Image src={HeadShot} alt="Head Shot of Freddie Hoy" />
    <AboutContentContact />
    <AboutContentInterests />
  </Container>
);
