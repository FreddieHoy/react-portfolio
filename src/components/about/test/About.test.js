import { shallow } from 'enzyme';
import React from 'react';

import { About } from '../About';

describe('About', () => {
  it('render the correct fields', () => {
    const wrapper = shallow(<About />);

    expect(wrapper).toMatchSnapshot();
  });
});
