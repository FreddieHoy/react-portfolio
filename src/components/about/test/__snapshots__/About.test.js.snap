// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About render the correct fields 1`] = `
<Container>
  <AboutTitle />
  <AboutContentMain />
  <About__Image
    alt="Head Shot of Freddie Hoy"
    src=""
  />
  <AboutContentContact />
  <AboutContentInterests />
</Container>
`;
